Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Valeri Karpov <valkar207@gmail.com>",
            "Saghm Rossi <saghmrossi@gmail.com>",
            "Kevin Yeh <kevinyeah@utexas.edu>"]
 
-description = "An experimental MongoDB driver written by MongoDB interns."
+description = "A native-Rust MongoDB driver. Originally written by MongoDB interns, the driver is an unofficial but still actively maintained project."
```

Comment:
perhaps some inspiration from the [Go driver page](https://docs.mongodb.com/ecosystem/drivers/go/) (community-driven) and [mongodb-labs description](https://github.com/mongodb-labs) ("from MongoDB Inc.")?

"A native MongoDB driver. Developed under MongoDB labs and maintained by the community."